src/view/pages/Farmer/tabs/Loan/index.tsx

```diff
@@ -53,6 +60,7 @@ class Buy extends React.Component<PropsType, OwnStateType> {
             <DataTable
               headers={['Date', 'Remaining Balance', 'Weekly Payment']}
               values={this.props.loanTransactions}
+              onEntryPress={this.onEntryPress}

```

Comment:
added all this stuff to stop it complaining. It will need to be put in place later anyway